cpp/modmesh/math/pymod/wrap_Complex.cpp

```diff
@@ -51,43 +52,52 @@ class MODMESH_PYTHON_WRAPPER_VISIBILITY WrapComplex
         : WrapBase<WrapComplex<value_type>, modmesh::Complex<value_type>, std::shared_ptr<modmesh::Complex<value_type>>>(mod, pyname, pydoc)
     {
         namespace py = pybind11; // NOLINT(misc-unused-alias-decls)
+        static const auto complex_dtype = py::dtype(std::string(std::is_same<value_type, float>::value ? "complex64" : "complex128"));
```

Comment:
I guess `std::string` can be removed in this line?

Reply:
I guess the use of ternary expression makes `std::string` necessary.  But it's good to double check.